### ISSUE TITLE
Enhancement: Publish MSIX version (Microsoft Store)

### ISSUE THREAD

**Issue by v3DJG6GL** [Betterbird/thunderbird-patches#285]
Hello everyone :)
First of all: Thank you very much for your work!

I often think about how nice it would be if there was an MSIX version of Betterbird - installable via winget or the Microsoft Store.
The main advantages would be **admin-less installation** possibilities and of course **automatic updates**.
This would make installation and maintenance much easier for all my friends and family ;)

Thunderbird is already available as MSIX, so it might be possible to build on this work:
https://apps.microsoft.com/detail/9pm5vm1s3vmq
https://bugzilla.mozilla.org/show_bug.cgi?id=1306911

Here are some docs for the MSIX version of Firefox. However, I have not found any similar documentation for Thunderbird.
https://firefox-source-docs.mozilla.org/browser/installer/windows/installer/MSIX.html

**Comment by Betterbird** [comment]
Winget is already available: https://winstall.app/apps/Betterbird.Betterbird

We have no plans for an MSIX installer.

**Comment by v3DJG6GL** [comment]
Yes, I know that Betterbird is available via winget, I install apps via winget most of the time.
But still: You don't have the advantages that you have with native MSIX installers: Installable without admin rights & automatic updates...

But anyway - thanks for your reply. I hope there is a point in the future where you provide a MSIX installer...

**Comment by Betterbird** [comment]
> & automatic updates...

How so? That will only happen if the product itself has an update function. I believe that winget already tracks updates.

**Comment by v3DJG6GL** [comment]
> How so? That will only happen if the product itself has an update function. I believe that winget already tracks updates.

you can also update non-MSIX apps with `winget update` yes.
But you would still need admin rights to install these updates because it's not a native MSIX app - e.g. KeePassXC which is also available at the Microsoft Store, although it's not a native MSIX app:
![image](https://github.com/Betterbird/thunderbird-patches/assets/72495210/9febca4a-39a9-4923-9123-7d0483b23df4)

With native MSIX apps it's more like F-Droid or Play Store on any Android Smartphone: Updates are automatically installed & managed by the package manager. The app itself doesn't need an integrated update function. One example is the Firefox fork LibreWolf which also has no integrated update function. But since it's available via Windows Store, updates are managed by the Windows Package Manager:
https://apps.microsoft.com/detail/9nvn9sz8kfd7
